ftmstore/store.py

```diff
@@ -20,7 +20,7 @@ def __init__(
         self.prefix = prefix
         self.database_uri = database_uri
         # config.setdefault('pool_size', 1)
-        self.engine = create_engine(database_uri, **config)
+        self.engine = create_engine(database_uri, future=True, **config)
```

Comment:
`requirements.txt` still requires 2.0.x. That seems unintended?

Reply:
`requirements.txt` isn't used then running `pip install`. It's usually used for local dev or when a `Docker` image specifically runs `pip install -r requirements.txt` for a repo. In my view, it's fine is the version of `SQLAlchemy` stays pinned in `requirements.txt`, especially if it's pinned to v2, since that's ultimately what we want all these libs to use.